libnmstate/schemas/operational-state.yaml

```diff
@@ -435,3 +419,23 @@ definitions:
                     type: boolean
                   state:
                     type: string
+
+  route:
+    type: object
+    properties:
+      state:
+        type: string
+        enum:
+          - absent
+      table-name:
+        type: string        # empty for unknown
+      table-id:
+        type: integer       # -1 means unknown
```

Comment:
It cannot be unknown as any route belongs to some table.

Reply:
Fixed by getting route table id from `NM.IPRoute.get_attribute()`.